checks.d/redisdb.py

```diff
@@ -325,6 +327,21 @@ def _check_slowlog(self, instance, custom_tags):
 
         self.last_timestamp_seen[ts_key] = max_ts
 
+    def _check_command_stats(self, conn, tags):
+        """Get command-specific statistics from redis' INFO COMMANDSTATS command
+        """
+        try:
+            command_stats = conn.info("commandstats")
+        except:
```

Comment:
fair enough, can just catch Exception, but we don't want anything that fails this process to fail the full agent check, right?